syntax = "proto3";

message ProcessExec {
  string file_path = 1;
}

message FileOpen {
  string file_path = 1;
}

message FileCreated {
  string file_path = 1;
}

message Event {
  string process_path = 1;
  oneof data {
    ProcessExec process_exec = 2;
    FileOpen file_open = 3;
    FileCreated file_created = 4;
  }
}
